src/main/java/com/esri/samples/scene/get_elevation_at_a_point/GetElevationAtAPointSample.java

```diff
@@ -0,0 +1,179 @@
+
+    @Override
+    public void start(Stage stage) {
+
+        try {
+            // create stack pane and application scene
+            StackPane stackPane = new StackPane();
+            Scene fxScene = new Scene(stackPane);
+            fxScene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+            // set title, size, and add JavaFX scene to stage
+            stage.setTitle("Get Elevation At A Point Sample");
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(fxScene);
+            stage.show();
+
+            // create a scene and add a basemap to it
+            ArcGISScene scene = new ArcGISScene();
+            scene.setBasemap(Basemap.createImagery());
+
+            // add the SceneView to the stack pane
+            sceneView = new SceneView();
+            sceneView.setArcGISScene(scene);
+            stackPane.getChildren().add(sceneView);
+
+            // add base surface for elevation data
+            Surface surface = new Surface();
+            surface.getElevationSources().add(new ArcGISTiledElevationSource("http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"));
+            scene.setBaseSurface(surface);
+
+            // create a point symbol to mark where elevation is being measured
+            SimpleLineSymbol lineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFFFF0000, 3.0f);
+
+            // create a text symbol to display the elevation of selected points
+            TextSymbol elevationTextSymbol = new TextSymbol(20,null,0xFFFF0000,TextSymbol.HorizontalAlignment.CENTER,TextSymbol.VerticalAlignment.MIDDLE);
+
+            // create a graphics overlay
+            graphicsOverlay = new GraphicsOverlay(GraphicsOverlay.RenderingMode.DYNAMIC);
+            graphicsOverlay.getSceneProperties().setSurfacePlacement(LayerSceneProperties.SurfacePlacement.RELATIVE);
+            sceneView.getGraphicsOverlays().add(graphicsOverlay);
+
+            // add a camera and initial camera position
+            Camera camera = new Camera(28.42, 83.9, 10000.0, 10.0, 80.0, 0.0);
+            sceneView.setViewpointCamera(camera);
+
+            // create listener to handle clicked
+            sceneView.setOnMouseClicked(event -> {
+
+                // get the clicked screenPoint
+                Point2D screenPoint = new Point2D(event.getX(), event.getY());
+                // convert the screen point to a point on the surface
+                Point relativeSurfacePoint = sceneView.screenToBaseSurface(screenPoint);
+
+                // check that the point is on the surface, and primary button was clicked
+                if (relativeSurfacePoint != null && event.getButton() == MouseButton.PRIMARY && event.isStillSincePress()) {
+
+                    // clear any existing graphics from the graphics overlay
+                    graphicsOverlay.getGraphics().clear();
```

Comment:
You could make two graphics -- one for the polyline and one for the text at the beginning -- and then reuse them instead of creating new ones every time.

To create a graphic without an initial geometry, do:
```java
Graphic graphic();
graphic.setSymbol(textSymbol);

//later
graphic.setGeometry(topOfPolyline);
```

By reusing, I mean just set the new geometry of each symbol, and the text for the text symbol.